diff --git a/libz/libz/zconf.h.in b/libz/libz/zconf.h.in
index 62adc8d..314b84b 100644
--- a/libz/libz/zconf.h.in
+++ b/libz/libz/zconf.h.in
@@ -433,11 +433,11 @@ typedef uLong FAR uLongf;
    typedef unsigned long z_crc_t;
 #endif
 
-#ifdef HAVE_UNISTD_H    /* may be set to #if 1 by ./configure */
+#if @HAVE_UNISTD_H@    /* may be set to #if 1 by ./configure */
 #  define Z_HAVE_UNISTD_H
 #endif
 
-#ifdef HAVE_STDARG_H    /* may be set to #if 1 by ./configure */
+#if @HAVE_STDARG_H@    /* may be set to #if 1 by ./configure */
 #  define Z_HAVE_STDARG_H
 #endif
 
